### DISCUSSION TITLE
Work of John Ehlers and Entropy

### DISCUSSION THREAD

**Discussion by cryptocoinserver** [DaveSkender/Stock.Indicators#604] [Ideas]
Have this repo starred with many of John Ehlers filters and some more indicators in C++. Might be easy to convert to C#.

[https://github.com/jerickson7/erickson_talib/tree/master/ta_library/lib](https://github.com/jerickson7/erickson_talib/tree/master/ta_library/lib)

For example:
- autocorr_periodogram
- entropy
- super_smoother
- roofing_filter
- high_pass_filter
- band_pass_filter
- decycler

**Comment by cryptocoinserver** [comment]
Some more resources that even are already in C#:

- [https://github.com/FxStill/FreeIndicators/tree/main/NinjaTrader%208/Indicators/Ehlers](https://github.com/FxStill/FreeIndicators/tree/main/NinjaTrader%208/Indicators/Ehlers)
- [https://github.com/westsider/indicatorsAug2021](https://github.com/westsider/indicatorsAug2021)
- [https://github.com/dynoweb/ninja-code/tree/master/NinjaTrader%207/bin/Custom/Indicator](https://github.com/dynoweb/ninja-code/tree/master/NinjaTrader%207/bin/Custom/Indicator)
- [https://github.com/LucidDion/TASC-Extensions/tree/master/TASCExtensions/TASCExtensions](https://github.com/LucidDion/TASC-Extensions/tree/master/TASCExtensions/TASCExtensions)
- [https://github.com/bizcad/LeanITrend/tree/master/Indicators/Custom%20Indicators](https://github.com/bizcad/LeanITrend/tree/master/Indicators/Custom%20Indicators)
- [https://github.com/QuantConnect/Lean/tree/master/Indicators](https://github.com/QuantConnect/Lean/tree/master/Indicators)

**Comment by DaveSkender** [comment]
Nice.  These look like good additions for the library.  If you want to track more closely, submit individually as [Issues](https://github.com/DaveSkender/Stock.Indicators/issues).  Any initial thoughts on which of these should go first?

**Reply by cryptocoinserver** [reply]
[https://github.com/LucidDion/TASC-Extensions/tree/master/TASCExtensions/TASCExtensions](https://github.com/LucidDion/TASC-Extensions/tree/master/TASCExtensions/TASCExtensions) seems to be based on the [Technical Analysis of STOCKS & COMMODITIES Magazine](http://traders.com/). Might be a good indicator for quality?

**Reply by DaveSkender** [reply]
Which one are you referring to?
